### DISCUSSION TITLE
Is there a way to skip connecting to wifi without pressing the Boot button?

### DISCUSSION THREAD

**Discussion by hakylee** [alf45tar/PedalinoMini#425] [Q&A]
I recently built the pedalino mini but it takes about 3mins to boot up to allow me to start using the controller without a wifi connection since it goes through all the process of trying to connect to wifi via provisioning via USB, smart config etc.

Is there a way to bypass all this (without pressing the Boot button) so that it only tries to connect to the last accessed wifi (for a few sec) and if it does not, it boots so that I can use the controller straight away?

**Comment by alf45tar** [comment]
Change the boot mode to Bluetooth only or to Access Point.
